README.md

```diff
@@ -21,13 +21,13 @@
 ## The Live Coding Music Synth for Everyone.
 
 Sonic Pi is a *new kind of musical instrument*. Instead of strumming
-strings or whacking things with sticks - you write code - **live**.
+strings or whacking things - you write code - **live**.
 
 Sonic Pi has been designed with the aim to find a harmonious balance
 between three core principles:
 
 * **Simple** enough for the 10 year old within you
-* **Joyful** enough for you to lose yourself through play
+* **Joyful** enough to lose yourself through play
```

Comment:
To me,  I feel that having the subject indicated before the action makes more sense, but I'll defer to @samaaron on this one